docs/index.md

```diff
@@ -39,11 +39,47 @@ Pulse provides push-based integrations with your workflow using the following al
 
 -   **One-click integrations**
 
-    Pulse currently supports the following "one-click" integrations that simplify the process of setting up your workflows to send data to Pulse:
-
-    -   **[GitHub integration](one-click-integrations/github-integration.md) or [Bitbucket integration](one-click-integrations/bitbucket-integration.md):** reports changes, deployments, and incidents
-    -   **[PagerDuty integration](one-click-integrations/pagerduty-integration.md):** reports incidents
-    -   **[Jira integration](one-click-integrations/jira-integration.md):** reports lead and cycle time
+    Pulse currently supports the "one-click" integrations listed in the table below that simplify the process of setting up your workflows to send data to Pulse.
```

Comment:
I really like this format!

Reply:
Indeed, great information presentation skills! :raised_hands: :nerd_face: